src/explore-education-statistics-admin/src/pages/admin-dashboard/components/__tests__/PublicationSummary.test.tsx

```diff
@@ -0,0 +1,332 @@
+    ).toBeInTheDocument();
+
+    const teamEmailLink = screen.getByText(testContact.teamEmail as string, {
+      selector: 'a',
+    });
+    expect(teamEmailLink).toBeInTheDocument();
+    expect(teamEmailLink).toHaveAttribute(
+      'href',
+      `mailto:${testContact.teamEmail}`,
+    );
+
+    expect(
+      screen.getByText(testContact.contactName as string),
+    ).toBeInTheDocument();
+
+    const contactTelNoLink = screen.getByText(
+      testContact.contactTelNo as string,
+      {
+        selector: 'a',
+      },
+    );
+    expect(contactTelNoLink).toBeInTheDocument();
+    expect(contactTelNoLink).toHaveAttribute(
+      'href',
+      `tel:${testContact.contactTelNo}`,
+    );
+  });
+
+  test('renders correct controls with full permissions', async () => {
+    render(
+      <MemoryRouter>
+        <PublicationSummary
+          publication={{
+            ...testPublication,
+            permissions: fullPermissions,
+          }}
+          topicId={testTopicId}
+          onChangePublication={noop}
+        />
+      </MemoryRouter>,
+    );
+
+    expect(screen.getByText('No releases created')).toBeInTheDocument();
+
+    expect(screen.getByText('Methodologies')).toBeInTheDocument();
+
+    expect(
+      screen.getByRole('button', {
+        name: 'Create methodology',
+      }),
+    ).toBeInTheDocument();
+
+    expect(
+      screen.getByRole('button', {
+        name: 'Link to an externally hosted methodology',
+      }),
+    ).toBeInTheDocument();
+
+    expect(
+      screen.getByText('Manage this publication', {
+        selector: 'a',
+      }),
+    ).toBeInTheDocument();
+
+    expect(
+      screen.getByText('Create new release', {
+        selector: 'a',
+      }),
+    ).toBeInTheDocument();
+  });
+
+  test('renders correctly with releases', async () => {
+    render(
+      <MemoryRouter>
+        <PublicationSummary
+          publication={{
+            ...testPublication,
+            releases: testReleases,
+          }}
+          topicId={testTopicId}
+          onChangePublication={noop}
+        />
+      </MemoryRouter>,
+    );
+
+    expect(screen.queryByText('No releases created')).not.toBeInTheDocument();
+
+    const releaseList = screen.getByTestId(
+      `Releases for ${testPublication.title}`,
+    );
+    expect(releaseList).toBeInTheDocument();
+
+    const releaseExpandButtons = within(releaseList)
+      .getAllByRole('button')
+      .filter(b => b.hasAttribute('aria-expanded'));
+
+    // We expect only 2 Releases to be visible here, as the "rel-3" Amendment is hiding the "rel-2" Release that
+    // it's an Amendment of.
+    expect(releaseExpandButtons).toHaveLength(2);
+
+    // Not exhaustively testing the contents of each Release, as this is handled in a separate component used by
+    // PublicationSummary.
+    const release1 = releaseExpandButtons[0];
+    expect(release1.textContent).toEqual(
+      `${testReleases[0].title} (not Live)Draft`,
```

Comment:
Think this highlights an issues with the tags. The text here basically illustrates what a screen reader sees, so it might end up sounding gibberish to a screen reader user.

We might want to add a space between the tag and the brackets.

Reply:
Done